src/modules/feedback/i18n/feedback.provider.js

```diff
@@ -27,7 +28,8 @@ export const provide = (lang) => {
 				mapFeedback_privacyPolicy: 'Hinweis zum Datenschutz',
 				mapFeedback_pleaseSelect: 'Bitte digitalisieren Sie mindestens einen Punkt als Ortsangabe für ihre Meldung.',
 				mapFeedback_disclaimer:
-					'Das LDBV behält sich grundsätzlich vor, Meldungen nicht zu übernehmen. Für evtl. Rückfragen, sowie zur Information über die weitere Bearbeitung, empfehlen wir die Angabe Ihrer E-Mail-Adresse.'
+					'Das LDBV behält sich grundsätzlich vor, Meldungen nicht zu übernehmen. Für evtl. Rückfragen, sowie zur Information über die weitere Bearbeitung, empfehlen wir die Angabe Ihrer E-Mail-Adresse.',
+				mapFeedback_geometry_missing: 'Eine Geometrie zum Feedback erstellen'
```

Comment:
`mapFeedback_pleaseSelect` can be removed, but 
`mapFeedback_geometry_missing` should have the value of `mapFeedback_pleaseSelect`